_sass/footer.scss

```diff
@@ -0,0 +1,29 @@
+#btnTop {
+    display: inline-block;
+    background-color: #0766CA; 
+    width: 50px;
+    height: 50px;
+    text-align: center;
+    border-radius: 4px;
+    position: fixed;
+    bottom: 30px;
+    right: 30px;
+    transition: background-color .3s, 
+      opacity .5s, visibility .5s;
+    opacity: 0;
+    visibility: hidden;
+    z-index: 1000;
+  }
+
+  #btnTop:hover {
+    cursor: pointer;
+    background-color: #333;
+  }
+  #btnTop:active {
+    background-color: #555;
+  }
+  #btnTop.show {
+    opacity: 1;
+    visibility: visible;
+  }
+  
```

Comment:
To get the smooth scrolling effect, just add the below css and it should do the trick.

```css
html {
    scroll-behavior: smooth;
}
```

Reply:
Wow, this was quite a simple solution. Everyone online keeps posting Jqueries for this. Also, is there a way to set the scroll speed? That would be an additional thing but would be quite helpful. Thanks a lot!

Reply:
I think jQuery would be needed to set the scroll speed, although I am not really sure if any css hack is present or not.

@louisefindlay23 do you think there is any css hack for scroll speed?

Reply:
Done some research. There are ways but they seem pretty complex.

This [vanilla JS snippet](https://www.sololearn.com/Discuss/1748649/how-to-change-scroll-speed) might work so you don't need the overhead of jQuery (if we're not already using it - not sure).

Not tested it though so YMMV. Hope it helps.